organization/_posts/2024/2024-08-29-coordination.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "HSF Coordination Meeting #273, 29 August 2024"
+layout: plain_toc
+---
+
+## Attending
+
+Present/Contributing: Graeme Stewart, Michel Jouvin, Stefan Roiser, Tommaso Lari
```

Comment:
I should be taken out of "present" and left below

Reply:
Done... and Paul added!